### PR TITLE
Release 0.13.0

### PR DESCRIPTION
**Pull request by aaronjudd**
This is a compatibility release aimed at updating Reaction with the rapid  progress in the Meteor platform. The biggest new feature in Meteor 1.3 is support for ES2015 modules on the client and the server.  Upcoming releases will continue a migration to a full ES2015 architecture.

- Meteor 1.3 compatibility release
- adds `meteor npm install` to Reaction installation workflow
- i18n enhancements and Russian language updates - @newsiberian
- UI improvements - @mikemurray
- Import improvements for merging Shop fixtures - @prinzdezibel
- new display of product titles in cart - @newsiberian
- improved payment processing handling - @zenweasel
- flat rate shipping handling updates - @newsiberian
- global layout updates handling
- many bug fixes, enhancements

Notes:
The Velocity project has been deprecated and now is no longer compatible with Meteor 1.3 CI testing, so testing is in a bit of flux.  We are working on a full migration to a new testing strategy to be implemented with the new modular structure refactor of Reaction.  The new modular release will be a Meteor 1.4+ compatible release.

### CODE REVIEW

File: imports/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+//
```
Comment (zenweasel) [inline imports/index.js]: Do we need this file or was this stuck here by the meteor update?

Comment (aaronjudd) [inline imports/index.js]: I think it's a refactor leftover, and that we don't need it.

File: packages/reaction-layout/client/templates/layout/admin/admin.js [hunk 2]
```diff
@@ -29,7 +30,7 @@ Template.coreAdminLayout.helpers({
           href: ReactionRouter.pathFor(shortcut.name),
           className: ReactionRouter.isActiveClassName(shortcut.name),
           icon: shortcut.icon,
-          tooltip: shortcut.label || "POOOOP??",
```
Comment (zenweasel) [inline packages/reaction-layout/client/templates/layout/admin/admin.js]: v13, now with less poop

Comment (aaronjudd) [inline packages/reaction-layout/client/templates/layout/admin/admin.js]: How is i18n being handled here?

**Comment by zenweasel** [comment]
Approved

**Pull request merged**